name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/csv/CSVSecurityExtractor.java

```diff
@@ -64,8 +61,12 @@ void extract(List<Item> items, String[] rawValues, Map<String, Column> field2col
                                             .toString()),
                             0);
 
-        // nothing to do to add the security: if necessary, the security item
-        // has been created in the callback of the #extractSecurity method
+        // The security item is created in the callback of the #getSecurity
+        // method, into this callback, symbols are not yet initialized, thus
+        // moves feed initialization from the callback to here by taking care of
+        // not overriding an existing value
+        if (security.getFeed() == null && security.getTickerSymbol() != null)
+            security.setFeed(YahooFinanceQuoteFeed.ID);
```

Comment:
By moving the change outside of the "getSecurity" this now applies to all existing securities as well. That can override the configuration that the user has chosen.

I have changed it to set the ticker also in the factory function above. It will be set again, but that should not hurt